docs/concepts/browsers-used-by-office-web-add-ins.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Web viewers used by Office Add-ins
+title: Browsers used by Office Add-ins
 description: ''
-ms.date: 05/03/2019
+ms.date: 05/23/2019
```

Comment:
Suggest that you populate the value for the `description` attribute, as doing so improves SEO.